baseplate/integration/pyramid.py

```diff
@@ -107,13 +152,21 @@ class BaseplateConfigurator(object):
 
     """
 
-    # TODO: remove the default on trust_trace_headers once apps are updated
     def __init__(self, baseplate, trust_trace_headers=False,
-                 edge_context_factory=None):
+                 edge_context_factory=None, header_trust_handler=None, secret_store=None):
         self.baseplate = baseplate
         self.trust_trace_headers = trust_trace_headers
```

Comment:
If all the trust logic is being moved to a ``header_trust_handler``, this doesn't need to be kept on the instance anymore.

Reply:
Can we just get rid of `self.trust_trace_headers`?  I'm guessing probably not?

Reply:
Would be nice, we could add a metric to see if anyone is using it